sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2021-05-24)
 
+- Updated the title and description of the package and updates to readme.
```

Comment:
1. I see 1.0.0-beta.2 is also 2021-05-04. So are we releasing two versions on the same day?
2. Usually change log is put into subsections like [key vault changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md). @g2vinay where is the template for change log?